test/api/test_rest.py

```diff
@@ -132,6 +133,23 @@ def test_project_builds(self):
         data = self.hit('/api/projects/%d/builds/' % self.project.id)
         self.assertEqual(3, len(data['results']))
 
+    def test_create_project_with_enabled_plugin_list(self):
```

Comment:
could you add a test that covers issue we found - 1 string in a list?

Reply:
sure. just updated the PR